python/radiance/__init__.py

```diff
@@ -0,0 +1 @@
+from radiance.light_output_node import *
```

Comment:
Should this be explicitly `... import LightOutputNode` or should `light_output_node.py` have `__all__ = ['LightOutputNode']`?